configure/using-cohorts.adoc

```diff
@@ -0,0 +1,16 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="using-cohorts"]
+= Using cohorts
+:context: using-cohorts
+
+toc::[]
+
+You can use cohorts to group cluster queues and determine which cluster queues are able to share borrowable resources with each other.
+Borrowable resources are defined as the unused nominal quota of all the cluster queues in a cohort.
+
+Using cohorts can help to optimize resource utilization by preventing under-utilization and enabling fair sharing configurations.
+Cohorts can also help to simplify resource management and allocation between teams, since you can group cluster queues for related workloads or for each team.
```

Comment:
Per ISG, [Do not use (since) as a synonym for because](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s).
```suggestion
Cohorts can also help to simplify resource management and allocation between teams, because you can group cluster queues for related workloads or for each team.
```

Reply:
Idk why but for some reason I thought it was the other way around 😂 ty